spec/support/factory_girl.rb

```diff
@@ -0,0 +1,7 @@
+RSpec.configure do |config|
+  config.include FactoryGirl::Syntax::Methods
```

Comment:
thoughts on adding this configuration directly in `spec_helper.rb`? that seems to be the convention we're leaning towards